target/zephyr/CMakeLists.txt

```diff
@@ -25,8 +25,12 @@ if(CONFIG_MENDER_MCU_CLIENT)
     zephyr_library_include_directories(${MENDER_PRIV_INCLUDE})
     zephyr_include_directories(${MENDER_MCU_INCLUDE})
 
-    file (STRINGS ${MENDER_MCU_VERSION_FILE} MENDER_CLIENT_VERSION)
-    zephyr_library_compile_definitions(-DMENDER_CLIENT_VERSION=\"${MENDER_CLIENT_VERSION}\")
+    # Include module from Zephyr CMake package. See:
+    # https://github.com/zephyrproject-rtos/zephyr/blob/v4.0.0/cmake/modules/git.cmake
+    include(git)
```

Comment:
Here I'd `include()` our file since we have it too. Who knows how they rename or change the file in the future.